src/django_components/dependencies.py

```diff
@@ -249,22 +248,11 @@ def set_component_attrs_for_js_and_css(
     if css_input_hash:
         all_root_attributes.append(f"data-djc-css-{css_input_hash}")
 
-    # These attributes are set on all tags
-    all_attributes = []
-
-    # We apply the CSS scoping attribute to both root and non-root tags.
-    #
-    # This is the HTML part of Vue-like CSS scoping.
-    # That is, for each HTML element that the component renders, we add a `data-djc-scope-a1b2c3` attribute.
-    # And we stop when we come across a nested components.
-    if css_scope_id:
```

Comment:
Removed mentions of css_scope_id from the codebase, since CSS scoping will be implemented as a separate extensions